index.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <link rel = "stylesheet"  type = "text/css" href = "style.css" />
+    <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
+    <title></title>
+  </head>
+  <body>
+    <header>
+            <img src="img/menu_bar.png" width="25" height="25" style="padding:17px;" />
+            <img src="img/youtube.png" width="100" height="64" />
+            <form class="search-form">
+              <input type="search" placeholder="search" >
+              <button type="submit" class="search-button">
+                <img src="img/search-icon.png"/>
+              </button>
+            </form>
+            <img src="img/account-icon.png" width="25" height="25" style="padding:17px;float:right;" />
+            <img src="img/bell-icon.png" width="25" height="25" style="padding:17px;float:right;" />
+            <img src="img/message-icon.png" width="40" height="25" style="padding:17px;float:right;" />
+            <img src="img/app-menu-icon.png" width="25" height="25" style="padding:17px;float:right;" />
+            <img src="img/camera-icon.png" width="25" height="25" style="padding:17px;float:right;" />
+
+    </header>
+    <div id="margine">
+
+    <div class="youtube_video">
+        <iframe width="100%" height="100%" src="https://www.youtube.com/embed/ZnuwB35GYMY" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
The width and height properties of the iframe use `px` as units. You only have to assign a digit. If you want to use a different unit try with css

Reply:
The frameborder attribute on the iframe element is obsolete. Use CSS instead.